pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -370,10 +372,24 @@ func validateUpdateEvent(e *event.UpdateEvent) bool {
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *PerformanceProfileReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	co, err := r.getClusterOperator()
+	if err != nil {
+		klog.Errorf("failed to get ClusterOperator: %v", err)
+		return reconcile.Result{}, err
+	}
+
+	operatorReleaseVersion := os.Getenv("RELEASE_VERSION")
```

Comment:
Is this maybe too strict? Well at least it allows changing the release version via the env if needed.

Reply:
Not sure I know what you mean here, but I'm assuming this is just a comment without any action needed.